packages/tbds-utilities/lib/inline-icon.scss

```diff
@@ -0,0 +1,25 @@
+$_icon-margin: .2em;
+$_visually-center-offset: -.1em;
+
+.tbds-inline-icon {
+  height: 1em;
+  position: relative;
+  top: $_visually-center-offset;
+  vertical-align: middle;
+
+  &:first-child {
+    margin-right: $_icon-margin;
+  }
+
+  &:last-child {
+    margin-left: $_icon-margin;
+  }
+}
+
+.tbds-inline-icon--large {
+  height: 1.25em;
```

Comment:
It feels like the `large` and `small` modifier values should be variables, and default, to allow consumers to tweak the sizing to their needs.